EXILED/Exiled.API/Features/Lockers/Locker.cs

```diff
@@ -91,6 +91,15 @@ public Locker(BaseLocker locker)
         /// </summary>
         public IReadOnlyCollection<Chamber> Chambers { get; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the locker is currently open.
+        /// </summary>
+        public bool IsOpen
+        {
+            get => Chambers.First().IsOpen;
+            set => Chambers.First().IsOpen = value;
+        }
+
```

Comment:
probably revert this change (except if an other dev boy thinks it's good enough too keep)